app/controllers/api/v1/doctors_controller.rb

```diff
@@ -20,8 +20,9 @@ def create
 
   def destroy
     @doctor = Doctor.find(params[:id])
+    temp = @doctor.id
     if @doctor.destroy
-      render json: 'Doctor deleted successfully'
+      render json: { message: 'Doctor deleted successfully', id: temp }
```

Comment:
I have done some experimentation on this will working on deletion from the front end, and you do not actually need to declare temp.

The line: `render json: { message:'Doctor deleted successfully', id: @doctor.id }` worked just fine for me, sending the id correctly even if @doctor was deleted.